StructureDefinition/Extension-DM-prescriptionType.StructureDefinition.xml

```diff
@@ -37,7 +37,7 @@
         <code value="Coding" />
       </type>
       <binding>
-        <strength value="required" />
+        <strength value="extensible" />
```

Comment:
Whats the reasoning behind this change? I presume so that the obsolete codes are valid because in code system, but not in the valueset?

Reply:
Exactly, we want to discourage the use of obsolete codes for new messages but allow them to pass validation as they will exist within V3 prescriptions. They are filtered out of the ValueSet through the filter but marking the binding strength as extensible means using the obsolete codes will result in a warning instead of an error